### ISSUE TITLE
Page to Blog macro creating blog posts as "Anonymous" on All Updates page

### ISSUE THREAD

**Issue by kberzinch** [chris-kent/Confluence-Blog-Tools#44]
**Describe the bug**

When "Publish Blog" is clicked on a page template with the "Page to Blog" macro, the blog post created appears to be created by "Anonymous" on the "All updates" home page of Confluence.

The creator is correct when viewing the page.

**To Reproduce**
1. Create a page template with the Page to Blog macro.
2. Use the template to create a blog post.
3. View the blog post on the "All updates" page

**Expected behavior**
The user that created the blog post would be displayed next to the entry.

**Screenshots**
![Untitled](https://user-images.githubusercontent.com/10482109/56143764-1764c780-5f6f-11e9-9326-3ed04fb71f99.png)

![Untitled2](https://user-images.githubusercontent.com/10482109/56143931-74607d80-5f6f-11e9-8cda-986840e0b592.png)
![Untitled1](https://user-images.githubusercontent.com/10482109/56143932-74607d80-5f6f-11e9-8f05-2e15a1f5c2aa.png)

**Versions (please complete the following information):**
 - Confluence: 6.15.2
 - Plugin: 4.0
 - Data Center: No

**Comment by chris-kent** [comment]
Hi,

I will be looking into this issue soon, sorry for the delay, I have been away on leave.

Cheers
Chris

**Comment by chris-kent** [comment]
Fixed in version 4.1

**Comment by erindelaneyb** [comment]
Hi Chris, 

Thanks for the fix. Is there something that can be done to remove the "Anonymous" credit for existing blog posts?

**Comment by chris-kent** [comment]
Hi Erin,

I don't believe there is an easy way, I found this article:
https://community.atlassian.com/t5/Confluence-questions/How-to-change-the-owner-author-of-a-page-in-Confluence/qaq-p/6734
it
is old and Atlassian have a feature improvement to add this, it may have
been implemented, I did not look at the issues reference on the page. But
there are some procedures to follow, some easy and some hard, depending on
what your outcome is to be (ie. the new creator name).

Hope this helps

Cheers
Chris
